aqueduct/gateway/tests/utils/base.py

```diff
@@ -126,6 +124,8 @@ def setUpClass(cls):
     def create_new_user() -> tuple[str, int]:
         # Create a new user and a new token for that user
         new_user = User.objects.create_user(username="OtherUser", email="other@example.com")
+        from django.contrib.auth.models import Group
```

Comment:
Do we get circular imports with this line? If not, I'd rather put it at the top of the module.